arm/Microsoft.AppConfiguration/configurationStores/deploy.bicep

```diff
@@ -102,6 +105,8 @@ param privateEndpoints array = []
 
 var enableReferencedModulesTelemetry = false
 
+var keyValuesList = !empty(keyValues) ? keyValues : []
```

Comment:
you can remove this var and directly act on the input array as suggested below
```suggestion
```

Reply:
Variable has been removed